template/Cargo.toml

```diff
@@ -19,6 +19,8 @@ lto = true
 [features]
 # If you uncomment this line, it will enable `wee_alloc`:
 #default = ["wee_alloc"]
+# If you uncomment this line, it will enable `console_error_panic_hook`:
+#default = ["console_error_panic_hook"]
```

Comment:
This may be a bit confusing, as you can't simply uncomment both lines to enable both of these at once.